Java/playwright/src/test/java/org/example/example_Test.java

```diff
@@ -0,0 +1,84 @@
+package org.example;
+import com.microsoft.playwright.*;
+import org.apache.commons.io.FileUtils;
+import org.junit.*;
+import com.deque.html.maven.axedevtools.playwright.AxePlaywrightBuilder;
+import com.deque.html.maven.axedevtools.utils.axeresults.AxeResults;
+import java.io.File;
+import java.io.IOException;
+
+
+public class example_Test {
+    static Playwright playwright = Playwright.create();
+    static Browser browser = null;
+    public static Page page = null;
+    public static AxePlaywrightBuilder axePlaywrightBuilder = null;
+    static String reporter = new File(
+            "src/test/resources/reporter-cli-macos")
+            .getAbsolutePath();
+    static String Logger = new File("axe-reports/").getAbsolutePath();
+
+    static String Destination = "a11y-results/";
+    static String command_html = reporter +" "+ Logger+" --destination "+Destination+" --format html";
+    static String command_csv = reporter +" "+ Logger+" --destination "+Destination+" --format csv";
+    static String command_xml = reporter +" "+ Logger+" --destination "+Destination+" --format xml";
+
+    @BeforeClass
+    public static void delete_json_dir() throws IOException {
+        browser = playwright.chromium().launch(new BrowserType.LaunchOptions().setHeadless(false).setSlowMo(50));
+        page = browser.newPage();
+        axePlaywrightBuilder = new AxePlaywrightBuilder(page);
+        File file = new File(Logger);
+        if (file.exists()){
+            FileUtils.deleteDirectory(file);
+        }
+    }
+
+    @Test
+    public void test1(){
+        page.navigate("https://broken-workshop.dequelabs.com/");
+        AxeResults axeResults = axePlaywrightBuilder.analyze();
+    }
```

Comment:
We are not asserting anything in this test, also the test name does not reflect what the test is doing: 
```suggestion
    @Test
    public void isBrokenWorshopAccessible(){
        page.navigate("https://broken-workshop.dequelabs.com/");
        AxeResults axeResults = axePlaywrightBuilder.analyze();
        
        assertTrue(axeResults.violationFree());
    }
```